.github/workflows/pinned-dependencies.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
         uses: sergeysova/jq-action@a3f0d4ff59cc1dddf023fc0b325dd75b10deec58 # v2.3.0
         id: versions
         with:
-          cmd: "jq '[getpath([\"dependencies\"],[\"devDependencies\"],[\"optionalDependencies\"])] | del(..|nulls) | map(.[]) | join(\",\")' package.json -r"
```

Comment:
are we good with this or should we remove?

Reply:
`yarn prettier` was complaining about it, not sure if this was a recent change for `prettier` or something else...

Reply:
I'll try reverting this and see if it's ok